### ISSUE TITLE
About learning rate strategy for LAPAR

### ISSUE THREAD

**Issue by wangyue7777** [Jia-Research-Lab/Simple-SR#6]
Hi, 

I notice that you use the cosine learning rate strategy to train the LAPAR network.
Would you please tell me the parameters you used, like the periods and the restart weight for the cosine learning rate strategy?

Thank you very much!

**Comment by fenglinglwb** [comment]
Hi, thanks for your interest. There are 3 training periods, each with 200K iterations. In each period, there is first a 2K warm up with a learning rate increasing from 4e-5 to 4e-4 linearly and then 198K iterations with a cosine weight decay.

**Comment by fenglinglwb** [comment]
The training code of LAPAR will be released in May.

**Comment by wangyue7777** [comment]
Hi,

Thanks for your reply! It helps a lot!

I just have one more question about the minimum learning rate for the cosine weight decay process.
Is it still 4e-5?

Thank you!

**Comment by fenglinglwb** [comment]
The minimum is 0.

wangyue7777 ***@***.***>于2021年4月30日 周五下午2:40写道：

**Comment by wangyue7777** [comment]
OK!

Thank you very much!
